SecretSanta/src/SecretSanta.Business/User.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SecretSanta.Business
+{
+    public class User
+    {
+        public int Id { get; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+#pragma warning disable INTL0001 // Fields _PascalCase ### current Naming is fine
+#pragma warning disable CA1051 // Do not declare visible instance fields ###can be public
+        public IList<Gift> Gifts = new List<Gift>();
```

Comment:
```suggestion
        public IList<Gift> Gifts ={ get; }
```
I think this is what you'd want here instead?